<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSaveAllVolumes</class>
 <widget class="QDialog" name="DialogSaveAllVolumes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save All Volumes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditOutputFolder">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output directory</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="toolButtonOpen">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditSuffix">
       <property name="text">
        <string>rotated</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Suffix to add</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBoxSaveToSourceDir">
       <property name="text">
        <string>Save to source directory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxNoResample">
     <property name="text">
      <string>Do not resample voxel data (only update header)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxCrop">
     <property name="text">
      <string>Crop to original volume size</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelErrorMessage">
     <property name="styleSheet">
      <string notr="true">color: rgb(255,50,50)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="freeview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogSaveAllVolumes</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSaveToSourceDir</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditOutputFolder</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSaveToSourceDir</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonOpen</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
